### PR TITLE
Colored indicator for afk or paused players

### PR DESCRIPTION
**Pull request by ardadem**

### CODE REVIEW

File: datasrc/network.py [hunk 1]
```diff
@@ -229,6 +229,8 @@
 	NetObjectEx("DDNetPlayer", "player@netobj.ddnet.tw", [
 		NetIntAny("m_Flags"),
 		NetIntRange("m_AuthLevel", "AUTHED_NO", "AUTHED_ADMIN"),
+                NetIntAny("m_PauseState"),
```
Comment (def-) [inline datasrc/network.py]: Any? Why not a range?

**Comment by def-** [comment]
In general: This only changes the color in scoreboard, people don't notice the scoreboard while playing. Wouldn't it be better to have an indication next to the tee?

**Comment by ardadem** [comment]
It's not change color in scoreboard. It's change color top of the tee which is very noticeable.

File: datasrc/network.py (hunk 1 above)
Comment (ardadem) [reply datasrc/network.py]: I set like this but it's simple to change it.

Comment (ardadem) [reply datasrc/network.py]: Done

**Comment by 12pm** [comment]
So many colors, maybe there is another way to mark afks

**Comment by fokkonaut** [comment]
put a label above the player's name, like the 0.7 ID next to the name but above it which says "AFK"

**Comment by Learath2** [comment]
I think I prefer a single color for anyone that wont be able to react, be
it pause or afk. I don't really like the idea of an AFK label on top

**Comment by ardadem** [comment]
How it's looks like right now?

**Comment by def-** [comment]
Why closed?

**Comment by ardadem** [comment]
Using text render didn't seem right

**Comment by Learath2** [comment]
I think it was fine with the text render, it's not so costly. Even if you were to render this differently, you'd have a hard time aligning that indicator any better as it's already aligned well, can't handle lowercase differently.

**Comment by ardadem** [comment]
Ok then i'll create new pr instead of reopen this. It's contains unnecessary changed commits.

**Pull request closed without merging**